nautobot_ssot/integrations/servicenow/servicenow.py

```diff
@@ -2,13 +2,13 @@
 
 import logging
 
+import requests  # pylint: disable=wrong-import-order
```

Comment:
```suggestion
import requests
```

Reply:
Do we as maintainers need to respond to these, or do either you or the drift management tool handle this?

Reply:
If you have any background on why this was implemented in this way then please leave a comment but I'll handle committing the change if we decide it's not going to break anything